docker/containers/docker_dev/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG SOURCE=ghcr.io/neuralmagic/cuda-python3.8
+# how were these versions selected before?
```

Comment:
likely latest supported at time of creation - or highest supported that worked out of the box with the CUDA image